### DISCUSSION TITLE
dump only specific path (s) of the tree

### DISCUSSION THREAD

**Discussion by DvoraNagler** [kayjan/bigtree#233] [Questions]
how can I dump only specific path (s) of the tree? 
Is it should be `print_tree(str_to_tree(leaf.path_name))` since this create one node only and not all the path from some reasons

**Comment by kayjan** [comment]
You can possibly explore [pruning the tree](https://bigtree.readthedocs.io/en/stable/bigtree/tree/helper/#bigtree.tree.helper.prune_tree) to that specific path - this will return a subset of the tree with all the existing attributes (if any).

In your example, when you perform `str_to_tree`, you are creating a new tree from the path and attribute information may be lost.
